app/views/categories/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= stylesheet_link_tag 'transactions.css', media: 'all', 'data-turbolinks-track': 'reload' %>
+<div class="container transaction-container">
+  <div class="container header">
+    <%= link_to authenticated_root_path do %>
+      <span class="fa fa-arrow-left"></span>
+    <% end %>
+    <h2>Transactions</h2>
+  </div>
+
+  <img src="<%=@category.icon%>" alt="Icon" class="category-icon">
+  <p class="category-name"><%=@category.name%></p>
+  <p class="category-details"><%=@category.details%></p>
+
+  <% @category_transactions.each_with_index do |transaction, index| %>
+    <div class="transaction-details">
+      <p class="transaction-numero">Transaction №<%= index + 1 %>41200032</p>
+      <p class="transaction-date">
+        <i class="fa fa-calendar"></i>
+<span><%= transaction.created_at.strftime('%d %B, %Y')%> at <%=transaction.created_at.strftime('%H:%M:%S')%></span>
+      </p>
+    </div>
+  <% end %>
```

Comment:
- [ ] [OPTIONAL] Please consider sticking to the design guidelines by ensuring that you follow the general layout, font-family, font-size, font-weight, and colors as well. The final design looks some way off of the original design. Please this is important! 👍 
### Design Guideline
![image](https://github.com/ismailmunyentwari9/transact_track/assets/46839548/95ff3c15-efa1-442c-b99c-82761b80a0d9)

### Your Design
![image](https://github.com/ismailmunyentwari9/transact_track/assets/46839548/a3ce30ad-1078-438f-a9a0-3ccecbbcefb0)